src/main/resources/tags/info/geyserconnect.tag

```diff
@@ -4,8 +4,10 @@ aliases: gc
 ---
 
 There is an **Unofficial** public [GeyserConnect](https://github.com/GeyserMC/GeyserConnect) instance running. You can connect to it via
-Address: `geyserconnect.net`
-Port: `19132`
+*Addresses:* 
+Americas: `us.geyserconnect.net`
```

Comment:
```suggestion
United States: `us.geyserconnect.net`
```
In my opinion sounds better

Reply:
If the country is specified for us.geyserconnect then why not specify it for eu.geyserconnect?

Reply:
"Americas" also includes South America and Canada though

Reply:
but it’s not americas the sub domain us “us” - United States. it’s what everyone is used to hear and people know whether they live close enough to the US to want to connect there or elsewhere

Reply:
In video games we often use:
-NA for North America
-EU for Europa
-AS for Asia
Maybe we could use these "standards of the industry"

Reply:
I would be more practical and use United States because that's where the server is hosting. I would then update Europe to the country it resides in - that way you know which country each is in and it's not necessarily a region assignment but a choice.

Reply:
We could just link to GReaper's website, which if I understand correctly is more in line with how we want to deal with unofficial instances in the future